packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -1188,6 +1199,49 @@ export default class Meeting extends StatelessWebexPlugin {
     this.setUpLocusInfoMeetingInfoListener();
     this.setUpLocusInfoAssignHostListener();
     this.setUpLocusInfoMediaInactiveListener();
+    this.setUpBreakoutsListener();
+  }
+
+  /**
+   * Set up the listeners for breakouts
+   * @returns {undefined}
+   * @private
+   * @memberof Meeting
+   */
+  setUpBreakoutsListener() {
+    this.breakouts.on(BREAKOUTS.EVENTS.BREAKOUTS_CLOSING, () => {
+      Trigger.trigger(
+        this,
+        {
+          file: 'meeting/index',
+          function: 'setUpBreakoutsListener'
+        },
+        EVENT_TRIGGERS.MEETING_BREAKOUTS_CLOSING,
+      );
+    });
+
+    this.breakouts.on(BREAKOUTS.EVENTS.MESSAGE, (messageEvent) => {
+      Trigger.trigger(
+        this,
+        {
+          file: 'meeting/index',
+          function: 'setUpBreakoutsListener'
+        },
+        EVENT_TRIGGERS.MEETING_BREAKOUTS_MESSAGE,
+        messageEvent
+      );
+    });
+
+    this.breakouts.on(BREAKOUTS.EVENTS.MEMBERS_UPDATE, () => {
+      Trigger.trigger(
+        this,
+        {
+          file: 'meeting/index',
+          function: 'setUpBreakoutsListener'
+        },
+        EVENT_TRIGGERS.MEETING_BREAKOUTS_UPDATE,
```

Comment:
one says members update(in the breakounts.on), and the other general breakouts update(in the trigger). Is this on purpose?

Reply:
Are you asking why this is the same event being triggered eventually? This is because the result is the same for the application developer. Something has changed in the breakouts and they need to go update their state. There was no point in propagating two distinct events outside of the meetings plugin for this.

Reply:
I see, I was just curious why we are listening on "member_update" and trigger "breakout_update" instead of "member_update". Is a bit confusing but since it's the same data then maybe it's fine? Would be nice to be documented.

Reply:
The current session sends member update events and those will still apply. This is for members of other sessions, sessions the user isn't in which is handled by the consuming app going over the members of those sessions. It felt overkill to submit individual member update events for each other breakout session and then force the consuming app to keep track of all of those listeners, given that they all come from a single roster update event.